bokehjs/src/coffee/api/typings/models/transforms.d.ts

```diff
@@ -0,0 +1,4 @@
+declare namespace Bokeh {
+  export interface Transform extends Model, ITransform {}
+  export interface ITransform extends IModel {}
+}
```

Comment:
Typings look good. Also, you should run `gulp examples` just to make sure everything compiles. Though examples don't use mappers directly, so there should be no failures.